components/css/grid/grid.css

```diff
@@ -0,0 +1,29 @@
+.grid-container {
+  display: grid;
+  gap: 1rem;
+  padding: 1rem;
+  color: white;
+  width: 100%;
+}
+.grid {
+  background-color: rgb(5, 5, 5);
+  padding: 0.5rem;
+}
+.grid-cols-2 {
+  grid-template-columns: 1fr 1fr;
+}
+.grid-cols-3 {
+  grid-template-columns: 1fr 1fr 1fr;
+}
+.grid-cols-4 {
+  grid-template-columns: 1fr 1fr 1fr;
+}
+.grid-rows-2 {
+  grid-template-rows: 1fr 1fr;
+}
+.grid-rows-3 {
+  grid-template-rows: 1fr 1fr 1fr;
+}
+.grid-rows-4 {
+  grid-template-rows: 1fr 1fr 1fr 1fr;
```

Comment:
Suggestion- Instead of writing "1fr" multiple times, you can use property grid-template-rows: repeat(4, 1fr);